test/unit/snowflake_config_test.js

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+const assert = require('assert');
+const snowflake = require('./../../lib/snowflake');
+const ErrorCodes = require('./../../lib/errors').codes;
+const Logger = require('./../../lib/logger');
+const GlobalConfig = require('./../../lib/global_config');
+
+const LOG_LEVEL_TAGS = require('./../../lib/logger/core').LOG_LEVEL_TAGS;
+
+describe('Snowflake Configure Tests', function ()
+{
+  var originalConfig;
+
+  before(function ()
+  {
+    originalConfig = {
+      logLevel: Logger.getInstance().getLevelTag(),
+      insecureConnect: GlobalConfig.isInsecureConnect(),
+      ocspFailOpen: GlobalConfig.getOcspFailOpen(),
+      jsonColumnVariantParser: GlobalConfig.jsonColumnVariantParser,
+      xmlColumnVariantParser: GlobalConfig.xmlColumnVariantParser,
+    }
+  });
+
+  after(function ()
+  {
+    snowflake.configure(originalConfig);
+  });
+
+  ///////////////////////////////////////////////////////////////////////////
+  //// Test invalid arguments                                            ////
```

Comment:
instead of comment let's wrap test cases in describe

Reply:
Ok